### ISSUE TITLE
Submission to GNU ELPA

### ISSUE THREAD

**Issue by jcs090218** [emacs-tree-sitter/treesit-fold#11]
It seems like Emacs-devel plans to merge this package into Emacs 30. 🤔
Therefore, we will need people who have contributed more than 15 LOC to complete the paperwork.

These are contributors that need to get the paperwork done.

- [x] @jcs090218
- [ ] @junyi-hou
- [ ] @Nidish96
- [ ] @samrjack
- [x] @DevelopmentCool2449
- [ ] @mattiasdrp 
- [ ] @akirak

For more information, please follow the thread at https://lists.gnu.org/archive/html/emacs-devel/2024-07/msg00001.html.

**Comment by mattiasdrp** [comment]
Maybe I missed it but I read the whole thread and couldn't find which paperwork I need to sign. I'm happy for you that your package is included in emacs :-)

**Comment by akirak** [comment]
@jcs090218 I have already finished the paperwork to GNU ELPA.

**Comment by samrjack** [comment]
@jcs090218 It's awesome that this is being included! I've been quite busy recently but I'll try to set aside some time tomorrow or this weekend to get any necessary paperwork taken care of.

**Comment by tpeacock19** [comment]
I have already filled out this paperwork a years ago. Not sure if I need to resubmit it?

**Comment by jcs090218** [comment]
> I have already filled out this paperwork a years ago. Not sure if I need to resubmit it?

No, you are good then! :D You just need to list everyone who needs to sign the paperwork.

**Comment by junyi-hou** [comment]
Sorry for the late reply. I was on vacation and this email got buried in my
inbox. Will do it this weekend.

**Comment by Nidish96** [comment]
I had already signed the FSF paper for another repository back in January. Do I need to do it separately for this also?

**Comment by jcs090218** [comment]
> I had already signed the FSF paper for another repository back in January. Do I need to do it separately for this also?

No, you are good then! Thank you for your response! :D

**Comment by mattiasdrp** [comment]
My assignment has finally been signed by the FSF, I'm all good :-)
